### PR TITLE
gRPC multithreading example

### PR DESCRIPTION
**Pull request by JamesNK**
gRPC multithreading example

### CODE REVIEW

File: aspnetcore/grpc/services.md [hunk 1]
```diff
@@ -328,6 +328,46 @@ A gRPC call ends on the server once the gRPC method exits. The following argumen
 
 If a gRPC method starts background tasks that use these types, it must complete the tasks before the gRPC method exits. Continuing to use the context, stream reader, or stream writer after the gRPC method exists causes errors and unpredictable behavior.
 
+In the following example, the server streaming method could write to the response stream after the call has finished:
```
Comment (wadepickett) [inline aspnetcore/grpc/services.md:331]:
```suggestion
In the following example, the server streaming method writes to the response stream after the call has finished:
```

File: aspnetcore/grpc/services.md [hunk 2]
```diff
@@ -328,6 +328,46 @@ A gRPC call ends on the server once the gRPC method exits. The following argumen
...
+            await responseStream.WriteAsync(new ExampleResponse());
+            await Task.Delay(TimeSpan.FromSeconds(1));
+        }
+    });
+
+    await PerformLongRunningWorkAsync();
+}
+```
+
+The solution is to await the write task before exiting the method:
```
Comment (wadepickett) [inline aspnetcore/grpc/services.md:350]:
```suggestion
For the previous example, the solution is to await the write task before exiting the method:
```

**Review by wadepickett — APPROVED** [review]
Looks good.  I made a suggestion for a very minor item that you could take or leave.  :)

File: aspnetcore/grpc/services.md (hunk 1 above)
Comment (JamesNK) [reply aspnetcore/grpc/services.md:331]: I'll leave it as "could write" because it's not certain what will happen.

**Pull request merged**